cypress/e2e/default/learn/challenges/projects.ts

```diff
@@ -0,0 +1,222 @@
+import { SuperBlocks } from '../../../../../config/certification-settings';
+
+interface Meta {
+  challengeOrder: string[][];
+}
+
+interface Solution {
+  contents: unknown;
+  fileKey: string;
+}
```

Comment:
```suggestion
interface File {
  contents: unknown;
  fileKey: string;
}

type Solution = File[];
```
A solution is an array of these types, so it's a little confusing if this is called "Solution".